src/Overlay/Dialog/Dialog.styled.ts

```diff
@@ -25,6 +43,8 @@ export const Dialog = styled.dialog`
     min-height: 100px;
     max-width: 85%;
     max-height: 85%;
+    width: ${({ $size }) => $size ? css` ${getWidthSize($size)}` : '200px'};
+    height: ${({ $size }) => $size ? css` ${getHeightSize($size)}` : '100px' };
```

Comment:
Maybe we shouldn't set the height as well. We are ending up with big empty spaces when there isn't much content. Use `maxHeight` instead and if the content overflows have a scroll bar

![image](https://github.com/ynput/ayon-react-components/assets/49156310/3ca81b8c-aa99-457f-88e2-9cbfe462d72d)

Reply:
I have given it maxHeight, whitespace is gone. Scrollbar is already there.
<img width="740" alt="image" src="https://github.com/ynput/ayon-react-components/assets/16327908/95e59caa-40dc-4d1c-9ea3-06bfd7f5789c">